### PR TITLE
Added lab 1 in python for Azure

### PR DESCRIPTION
**Pull request by Sealjay**
Created lab 1 in Python for Azure.

### CODE REVIEW

**Review by stack72 — COMMENTED** [review]
This is fantastic!!! Thank you so much :) I will review it asap!

**Comment by Sealjay** [comment]
Brilliant :) mostly just a like for like with the csharp repo, barring some minor changes.

Any feedback appreciated!

File: labs/azure/python/01-iac/02-configuring-azure.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+# Configuring Azure
+
+Now that you have a basic project, let's set up your environemnt, and configure Azure support for it.
+
+## Step 1 &mdash; Create a virtual environment and install python dependencies
+Creation of a virtual environment can vary between operating systems - see the [Azure Quickstart](https://www.pulumi.com/docs/get-started/azure/review-project/) for more information.
+
+### Windows
```
Comment (stack72) [inline labs/azure/python/01-iac/02-configuring-azure.md:8]: I love this!! I need to back port this to the other Python labs for sure :D)

**Review by stack72 — APPROVED** [review]
This is fantastic. Thank you so much :)

**Pull request merged**